elytron/src/test/resources/org/wildfly/extension/elytron/mycredstore.cs

```diff
@@ -0,0 +1,3 @@
+# Properties Credential Store (Do Not Modify)
+# Generated with Elytron Tool Credential Store. Secret Key Credential Store
```

Comment:
Is there a specific command that was used?

Reply:
Sorry I was mistaken this was created using the jboss-cli.sh commands. Updated comment with command used